### PR TITLE
All hands page

### PR DESCRIPTION
**Pull request by m-ou-se**

### CODE REVIEW

File: src/pages/all-hands.astro [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+            The All-Hands Planning Team (to be established soon) will coordinate the schedules and room assignments in advance,
+            to maximize the opportunities for collaboration between teams.
+            </p>
+            <p>
+            We will have 12 rooms available to us: 8 small rooms (~12 seats), 4 slightly bigger rooms (~14 seats),
+            a medium sized room (~20 seats), a large square room (~40 seats), a long rectangular room (~48 seats),
+            and a big room that fits all of us (~200 seats in a theatre layout).
+            </p>
+            <p>
+            Each room will have tables made out of cartboard that you can write on,
```
Comment (redshiftss) [inline src/pages/all-hands.astro]: Should be cardboard instead of cartboard

File: src/pages/all-hands.astro [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+            a whiteboard and/or flipover, and a large hdmi screen.
```
Comment (redshiftss) [inline src/pages/all-hands.astro]: HDMI should be capitalized

File: src/pages/all-hands.astro [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+            </p>
+            <p>
+            To attend the conference, you'll need a <a href="https://www.eventbrite.nl/e/rust-week-2025-tickets-1001676961757?aff=oddtdtcreator">conference ticket</a>.
+            If your employer is unable to pay for your ticket, you can request the Rust Foundation to pay for your ticket by contacting grants@rust-foundation.org.
+            </p>
+        </div>
+        <div class="md:col-start-3 md:col-end-11 xl:col-start-4 xl:col-end-10" >
+            <h2>Pre-All-Hands (Thursday)</h2>
+            <p>
+            The venue of the all-hands will open a day early for an optional and informal "pre-all-hands" day.
```
Comment (redshiftss) [inline src/pages/all-hands.astro]: all-hands should be capitalized consistently

**Comment by jdonszelmann** [comment]
lgtm

**Pull request merged**